### PR TITLE
Doc improve

### PR DESCRIPTION
**Pull request by wontheone1**
After using c-api for the first time, there were few points I was not sure how to do it. Or I found examples deep inside repository but thought I would have saved some times if the examples were in the README. This PR aims to address those problems that library first time users might encounter.

Thank you metosin for making awesome libraries like this one.

### CODE REVIEW

**Comment by wontheone1** [comment]
@ikitommi Hi, have you had time to check the PR? I think this might help completely new users to library a bit (it would have helped me). If you don't like some parts, you can give me feedback, or if this change is not necessary, we can close the PR :)

**Comment by ikitommi** [comment]
Thanks for the PR! Few comments in.

File: README.md [hunk 1]
```diff
@@ -47,9 +47,26 @@ See [CHANGELOG](https://github.com/metosin/compojure-api/blob/master/CHANGELOG.m
     (ok {:message (str "Hello, " name)})))
 ```
 
+### Validating an API
```
Comment (ikitommi) [inline README.md]: The old README had all the details in it and it was really hard to read. I think validation is a details that should be in the wiki, not in README. Could you move this part to wiki?

File: README.md [hunk 2]
```diff
@@ -47,9 +47,26 @@ See [CHANGELOG](https://github.com/metosin/compojure-api/blob/master/CHANGELOG.m
...
+
+```clj
+(require '[compojure.api.sweet :refer :all])
+(require '[compojure.api.validator :as v])
+(require '[midje.sweet :refer :all])
+(require '[ring.util.http-response :refer :all])
+
+(fact
+  (v/validate (defapi app
```
Comment (ikitommi) [inline README.md]: `api` should be used here. `defapi` will be deprecated in the 2.0.0

**Comment by ikitommi** [comment]
Thanks, merged. I'll move the validation to wiki.

**Comment by ikitommi** [comment]
Actually, the validation is in the wiki, need to reorganize for 2.0.0: https://github.com/metosin/compojure-api/wiki/Swagger-integration#compojure-api-validator

**Pull request merged**